lib/cisco_node_utils/cmd_ref/stp_global.yaml

```diff
@@ -7,7 +7,7 @@ _template:
   get_command: "show running-config spanning-tree"
 
 bd_designated_priority:
-  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
+  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k-EX, N9k]
```

Comment:
Are you planning to add `N9k-EX` as a recognized platform?

Reply:
yes